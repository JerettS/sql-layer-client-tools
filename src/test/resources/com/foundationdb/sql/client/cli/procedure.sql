CREATE PROCEDURE bang(IN n INT, OUT s VARCHAR(1024))
  LANGUAGE javascript PARAMETER STYLE variables AS $$
  s = "";
  while (n-- > 0) s += "!";
$$;

CALL bang(5);
