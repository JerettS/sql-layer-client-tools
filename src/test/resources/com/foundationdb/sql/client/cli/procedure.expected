ROWS: 0

   s
-------
 !!!!!
(1 row)

